### ISSUE TITLE
AWS EC2 windows server 2019 instance

### ISSUE THREAD

**Issue by estrandb** [StunlockStudios/vrising-dedicated-server-instructions#50]
Crashes with this error. Is a GPU required? Can you please provide required runtime libs and/or software packages needed to run the server?

ERROR: Shader GUI/Text Shader shader is not supported on this GPU (none of subshaders/fallbacks are suitable)

**Comment by crombieplex** [comment]
Im not sure about AWS but I have it running just fine on a windows 10 build with vultr.

**Comment by MiguelMachado-dev** [comment]
Yeah, I tried creating one at GCP but got some missing DLLs, would be great to have utils with all libs needed

**Comment by fishd72** [comment]
I'm running in a Windows 10 Pro VM on Azure without issue, there's no GPU presented to that system. I get the following entries in my server log, but it doesn't seem to present any problems for the server backend to run:

> Forcing GfxDevice: Null
GfxDevice: creating device client; threaded=0
NullGfxDevice:
    Version:  NULL 1.0 [1.0]
    Renderer: Null Device
    Vendor:   Unity Technologies
FMOD initialized on nosound output
Microsoft Media Foundation video decoding to texture disabled: graphics device is Null, only Direct3D 11 and Direct3D 12 (only on desktop) are supported for hardware-accelerated video decoding.
WARNING: Shader Unsupported: 'Shader Graphs/Effect' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
WARNING: Shader Unsupported: 'Shader Graphs/Effect' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
WARNING: Shader Unsupported: 'Shader Graphs/Shader_MasterLit' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
WARNING: Shader Unsupported: 'Shader Graphs/Shader_MasterLit' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
WARNING: Shader Unsupported: 'Shader Graphs/EffectLit' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
WARNING: Shader Unsupported: 'Shader Graphs/EffectLit' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
WARNING: Shader Unsupported: 'Shader Graphs/Liquid' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
WARNING: Shader Unsupported: 'Shader Graphs/Liquid' - All subshaders removed
WARNING: Shader Did you use #pragma only_renderers and omit this platform?
WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?

**Comment by manuva** [comment]
> I'm running in a Windows 10 Pro VM on Azure without issue, there's no GPU presented to that system. I get the following entries in my server log, but it doesn't seem to present any problems for the server backend to run:
> 
> > Forcing GfxDevice: Null
> > GfxDevice: creating device client; threaded=0
> > NullGfxDevice:
> > Version:  NULL 1.0 [1.0]
> > Renderer: Null Device
> > Vendor:   Unity Technologies
> > FMOD initialized on nosound output
> > Microsoft Media Foundation video decoding to texture disabled: graphics device is Null, only Direct3D 11 and Direct3D 12 (only on desktop) are supported for hardware-accelerated video decoding.
> > WARNING: Shader Unsupported: 'Shader Graphs/Effect' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
> > WARNING: Shader Unsupported: 'Shader Graphs/Effect' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
> > WARNING: Shader Unsupported: 'Shader Graphs/Shader_MasterLit' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
> > WARNING: Shader Unsupported: 'Shader Graphs/Shader_MasterLit' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
> > WARNING: Shader Unsupported: 'Shader Graphs/EffectLit' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
> > WARNING: Shader Unsupported: 'Shader Graphs/EffectLit' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
> > WARNING: Shader Unsupported: 'Shader Graphs/Liquid' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?
> > WARNING: Shader Unsupported: 'Shader Graphs/Liquid' - All subshaders removed
> > WARNING: Shader Did you use #pragma only_renderers and omit this platform?
> > WARNING: Shader If subshaders removal was intentional, you may have forgotten turning Fallback off?

Whats your server config?
